wdl/tasks/QC/AlignedMetrics.wdl

```diff
@@ -195,323 +183,7 @@ task MosDepth {
-        docker:                 select_first([runtime_attr.docker,            default_attr.docker])
-    }
-}
-
-task FilterMQ0Reads {
-    input {
-        File bam
-
-        RuntimeAttr? runtime_attr_override
-    }
-
-    Int disk_size = 2*ceil(size(bam, "GB"))
-    String prefix = basename(bam, ".bam")
-
-    command <<<
-        set -euxo pipefail
-
-        samtools view -q 1 -b ~{bam} > ~{prefix}.no_mq0.bam
-        samtools index ~{prefix}.no_mq0.bam
-    >>>
-
-    output {
-        File no_mq0_bam = "~{prefix}.no_mq0.bam"
-        File no_mq0_bai = "~{prefix}.no_mq0.bam.bai"
-    }
-
-    #########################
-    RuntimeAttr default_attr = object {
-        cpu_cores:          1,
-        mem_gb:             2,
-        disk_gb:            disk_size,
-        boot_disk_gb:       10,
-        preemptible_tries:  2,
-        max_retries:        1,
-        docker:             "us.gcr.io/broad-dsp-lrma/lr-metrics:0.1.11"
-    }
-    RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
-    runtime {
-        cpu:                    select_first([runtime_attr.cpu_cores,         default_attr.cpu_cores])
-        memory:                 select_first([runtime_attr.mem_gb,            default_attr.mem_gb]) + " GiB"
-        disks: "local-disk " +  select_first([runtime_attr.disk_gb,           default_attr.disk_gb]) + " HDD"
-        bootDiskSizeGb:         select_first([runtime_attr.boot_disk_gb,      default_attr.boot_disk_gb])
-        preemptible:            select_first([runtime_attr.preemptible_tries, default_attr.preemptible_tries])
-        maxRetries:             select_first([runtime_attr.max_retries,       default_attr.max_retries])
-        docker:                 select_first([runtime_attr.docker,            default_attr.docker])
-    }
-}
-
-task ComputeBedCoverage {
-    input {
-        File bam
-        File bai
-        File bed
-        String prefix
-
-        RuntimeAttr? runtime_attr_override
-    }
-
-    Int disk_size = 2*ceil(size(bam, "GB") + size(bai, "GB") + size(bed, "GB"))
-
-    command <<<
-        set -euxo pipefail
-
-        bedtools coverage -b ~{bed} -a ~{bam} -nobuf | gzip > ~{prefix}.txt.gz
-        zcat ~{prefix}.txt.gz | awk '{ sum += sprintf("%f", $15*$16) } END { printf("%f\n", sum) }' > ~{prefix}.count.txt
-    >>>
-
-    output {
-        File coverage = "~{prefix}.txt.gz"
-        Float counts = read_float("~{prefix}.count.txt")
-        File counts_file = "~{prefix}.count.txt"
-    }
-
-    #########################
-    RuntimeAttr default_attr = object {
-        cpu_cores:          1,
-        mem_gb:             2,
-        disk_gb:            disk_size,
-        boot_disk_gb:       10,
-        preemptible_tries:  2,
-        max_retries:        1,
-        docker:             "us.gcr.io/broad-dsp-lrma/lr-metrics:0.1.11"
+        docker:             "us.gcr.io/broad-dsp-lrma/lr-mosdepth:0.3.2"
```

Comment:
Similarly, mosdepth's docker is being updated. Can we look into what changed in 0.3.2?

Reply:
I just added a couple of packages so that we didn't have to needlessly break this task in two.